mmv1/products/networkservices/EndpointPolicy.yaml

```diff
@@ -0,0 +1,152 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'EndpointPolicy'
+base_url: 'projects/{{project}}/locations/global/endpointPolicies'
+create_url: 'projects/{{project}}/locations/global/endpointPolicies?endpointPolicyId={{name}}'
+self_link: 'projects/{{project}}/locations/global/endpointPolicies/{{name}}'
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  EndpointPolicy is a resource that helps apply desired configuration on the endpoints that match specific criteria.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1beta1/projects.locations.endpointPolicies'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+import_format: ['projects/{{project}}/locations/global/endpointPolicies/{{name}}']
```

Comment:
```suggestion
```

What happens if we omit this? I think it's redundant with `self_link`.

Reply:
You are right, I didnt get any test failures, so removed this.